src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
+KeyPair keypairForSignature (KeyType keyType, RippleAddress const& seed);
```

Comment:
Now I finally understand what `keypairForSignature` does. The name is quite confusing, who picked it? Anyway, in light of this use case we probably need to add
`static std::pair<AnySecretKey, AnyPublicKey> AnySecretKey::make_key_pair(KeyType type, <Seed> seed)`
I'm not entirely sure what type to use for `<Seed>` but obviously we'd prefer not to use `RippleAddress` there. Maybe it should be `uint256`.